src/builder.ts

```diff
@@ -27,31 +26,37 @@ interface FieldPublisherConfig {
   ordering?: boolean | Record<string, boolean>
 }
 
+export interface Options {
+  photon: (ctx: any) => any
+  photonPath?: string
+}
+
+export function build(options: Options) {
+  const builder = new NexusPrismaBuilder(options)
+  return builder.build()
+}
+
 export class NexusPrismaBuilder {
   protected readonly dmmf: DMMFClass
   protected visitedInputTypesMap: Record<string, boolean>
   protected whitelistMap: Record<string, string[]>
   protected argsNamingStrategy: IArgsNamingStrategy
   protected fieldNamingStrategy: IFieldNamingStrategy
 
-  constructor(protected params: NexusPrismaParams) {
-    let transformedDMMF
-
-    if (process.env.NEXUS_PRISMA_DEBUG) {
-      // Using eval so that ncc doesn't include it in the build
-      transformedDMMF = transformDMMF(eval(`require('@generated/photon'`).dmmf)
-    } else {
-      // @ts-ignore
-      transformedDMMF = __DMMF__
-    }
-
-    this.dmmf = new DMMFClass(transformedDMMF as any)
+  constructor(protected options: Options) {
+    // TODO Default should be updated once resolved:
+    // https://github.com/prisma/photonjs/issues/88
+    // TODO when photon not found log hints of what to do for the user
+    // TODO DRY this with same logic in typegen
+    const photonPath = options.photonPath || '@generated/photon'
+    const transformedDMMF = transformDMMF(require(photonPath).dmmf)
```

Comment:
Once we remove the exported dmmf from photon, this step will be too expensive to be viable in production, especially in lambda's

Reply:
Discussion between @Weakky and I since this comment was written contributed to 

https://www.notion.so/prismaio/Nexus-prisma-as-a-regular-library-178b5232def24800ad4c7e17fcc1201b

Conclusion was that we can rely on dmmf via photon or via its an import in its own right, either way though, the key, an import.